src/components/App/Post/PostQuickAdd.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { uniqueId } from 'lodash';
+
+import { Input } from '../../Form';
+
+const generatePostUrl = () => {
+  const imgNumber = Math.floor(Math.random() * 100);
+  return `https://picsum.photos/320/240?image=${imgNumber}`;
+};
+
+class PostQuickAdd extends Component {
+  emptyPost = {
+    title: '',
+    body: '',
+  };
+
+  state = {
+    ...this.emptyPost,
+  };
+
+  handleChange = ({ target: { value, name } }) => {
+    this.setState({ [name]: value });
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
+
+    this.props.onSubmit({
+      ...this.state,
```

Comment:
This can be unsafe in some cases, e.g. when you add something else to a state and then it's passed here. Imho nothing wrong here just need to be careful when extending this component's state.

Reply:
With great power, comes great responsibility :-) So true here.

I was asked why we dont use spread, thats why I used it here. BUT better solution would be to extract constant from state like `const {title, body} = this.state;` and then add those.